mediacapture-streams/MediaStream-MediaElement-preload-none-srcObject.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<!-- Copyright © 2016 Chromium authors and World Wide Web Consortium, (Massachusetts Institute of Technology, ERCIM, Keio University, Beihang). -->
+<html>
+    <head>
+        <title>Test that the HTMLMediaElement preload 'none' attribute value is ignored for MediaStream used as srcObject.</title>>
+        <link rel="author" title="Matthew Wolenetz" href="mailto:wolenetz@chromium.org"/>
+        <script src="/resources/testharness.js"></script>
+        <script src="/resources/testharnessreport.js"></script>
+        <script src="/common/vendor-prefix.js" data-prefixed-objects='[{"ancestors":["navigator"], "name":"getUserMedia"}, {"ancestors":["window"], "name":"MediaStream"}]'></script>
+    </head>
+    <body>
+        <p class="instructions">When prompted, accept to share your audio stream.</p>
+        <h1 class="instructions">Description</h1>
+        <p class="instructions">This test checks that the HTMLMediaElement preload 'none' attribute value is ignored for MediaStream used as srcObject.</p>
+
+        <audio id="aud"></audio>
+
+        <script>
+            var aud = document.getElementById("aud");
+            aud.preload="none";
+            var t = async_test("Tests that the HTMLMediaElement preload attribute value 'none' is ignored for MediaStream used as srcObject.");
```

Comment:
I agree lots of duplication. Rather than 3 files (2 html + 1 js), I'll merge it all into one html and refactor out a utility test function in that same html. If later we find that mediacapture-streams needs that utility elsewhere, a separate js could be used then.